release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -180,6 +180,35 @@ For more information, see xref:../post_installation_configuration/cluster-capabi
 [id="ocp-4-11-web-console"]
 === Web console
 
+[id="ocp-4-11-dynamic-plug-in-updates"]
+==== Dynamic plug-in updates
+
+With this update, you can now enable and disable plug-ins in the *Configuration* tab of the *Administration* -> *Cluster Settings* page of the web console. You can also add different extension types to customize your plug-in.
```

Comment:
Relates to #47507

Reply:
enable and disable dynamic plugins in Configuration tab has been introduced since 4.10, so do we want to highlight again in 4.11?

Reply:
@yapei should we remove this release note then since the extensions were also available in 4.10? I still plan to update the main docs.

Reply:
^ I spoke with @jhadvig and decided to add a note about `console.openshift.io/default-i18next-namespace` that was introduced in 4.11. @yapei could I have an ack for this note? Thank you!